app/controllers/company_settings_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CompanySettingsController < ApplicationController
+
+  #/company_settings/new
+  def new
+    @nc_settings = CompanySetting.new;
```

Comment:
what is nc ? i asked you all to name variables properly 
like here @company_settings